subprojects/test-runner/client/src/test/kotlin/com/avito/test/ListWithDefault.kt

```diff
@@ -0,0 +1,5 @@
+package com.avito.test
+
+internal fun <T : Any> listWithDefault(size: Int, value: T): List<T> {
+    return (0 until size).map { value }
```

Comment:
```suggestion
    return List(size) { value }
```

Reply:
done in next PR, removed this extension completely in favour of stdlib fun